ci/Dockerfile.ledger

```diff
@@ -0,0 +1,9 @@
+# Taken from bitcoindevkit/rust-hwi
+FROM ghcr.io/ledgerhq/speculos
+
+RUN apt-get update
+RUN apt-get install wget -y
+RUN wget "https://github.com/LedgerHQ/speculos/blob/master/apps/nanos%23btc%232.1%231c8db8da.elf?raw=true" -O /speculos/btc.elf
+ADD automation.json /speculos/automation.json
+
+ENTRYPOINT ["python", "./speculos.py", "--automation", "file:automation.json", "--display", "headless", "--vnc-port", "41000", "btc.elf"]
```

Comment:
This is really nice that we can have ci testing with docker for devices. Is it possible to have such docker for other major vendor like coldcard and trezor?

Reply:
Yes indeed! We do test with trezor emulator on rust-hwi (and coldcard is work-in-progress). I didn't want to bog down BDK tests, as the emulators take a lot of time to build and run, hence I thought we would do more testing on rust-hwi and then just test one device with BDK.